doc/source/index.rst

```diff
@@ -1,98 +1,68 @@
-****************************
-scikit-image |version| docs
-****************************
+scikit-image's documentation
+============================
 
-scikit-image is an image processing toolbox for SciPy_. View the
-`release notes <release_notes.html>`_ for |version|.
+**Date**: |today|, **Version**: |version|
 
-.. _SciPy: https://www.scipy.org
+Welcome! `scikit-image`_ is an image processing toolbox which builds on
+`numpy`_, `scipy.ndimage`_ and other libraries to provide a versatile set of
+image processing routines in `Python`_.
 
+.. _scikit-image: https://scikit-image.org
+.. _numpy: https://numpy.org
+.. _scipy.ndimage: https://docs.scipy.org/doc/scipy/reference/ndimage.html
+.. _Python: https://www.python.org
 
-Sections
-========
+.. grid:: 2
+    :gutter: 4
```

Comment:
> I prefer our previous landing page: it made it very easy to get to where you needed to be without any noise. Now, there's a lot of text to parse first, and the options are reduced. WDYT? -- @stefanv 

For the same reason I tried to reduce the number of shortcuts. I think a few don't necessarily have to be top-level. This is my initial suggestion at a compromise between what we had previously and what SciPy, and NumPy are using.

However, I there is probably room for improvement! I'll see if I can come up with something that makes everyone happy. 

In the end personal taste factors into this heavily. I think there are a few places were a bit of attention would make it a lot easier to find relevant information and it's really tempting to go for it. But I'm happy to keep keep this PR more focused on the theme for now. :wink: